script/docker-run.sh

```diff
@@ -31,23 +33,32 @@ fi
 # Checking image already has a running container
 CONTAINER_ID=$(docker ps -qf ancestor="$BIN_IMAGE_NAME")
 if [ ! $CONTAINER_ID = "" ]; then
-    echo "[ERR ] Container with ID '$CONTAINER_ID' already running for image '$BIN_IMAGE_NAME'"
+    echo "[ERR ] container with ID '$CONTAINER_ID' already running for image '$BIN_IMAGE_NAME'"
     exit 1
 fi
 
-PORT=$(jq '.port' "$CONF_FILE")
+PORT=$(cat "$CONF_FILE" | docker run -i imega/jq '.port')
```

Comment:
You are hereby condemned to burn in hell as long as you will run the (all)mighty `jq` in Docker.

Reply:
:D
I agree `jq` is (somehow)mighty, that is the reason I use it. I wish it was as standard as `sed` or `awk` so I would not have to do this